content/04.study.md

```diff
@@ -47,48 +47,54 @@ Deep learning applied to gene expression data is still in its infancy, but the f
-In addition to this, Gene expression has been used to infer and impute methylation states [@tag:Peng2019] [@tag:Levy-Jurgenson2018], methylation of genes predicted from promoter methylation [@tag:Pan2018], and convolutional models have been able to predict methylation status from images [@tag:Momeni2018] [@tag:Korfiatis2017].
-While these examples of methylation imputation and inference methods have value it is imperative to recognize limitations of imputing cytosine modifications.
-Imputing DNA methylation has complexities above and beyond genotype imputation: correlation of DNA methylation marks can depend on cell types and other factors that can vary by sample.
-As the number of tissue types and cell types with whole-genome bisulfite sequencing (and oxidative bisulfite sequencing) grows, the accuracy of DNA methylation imputation is expected to increase.
-While these methods reduce the computational overhead at comparable performance to other popular methylation imputation methods such as K-Nearest Neighbors, Random Forest, Singular Value Decomposition and Multiple Imputation by Chained Equations, the software implementations will need to become more user-friendly to gain widespread adoption.  
+DeepSignal employs a CNN to construct features from raw electrical Nanopore signals from sites near a methylated base.
+It uses a bidirectional RNN on DNA sequences of the aligned signals to detect methylation [@tag:Ni2018].
+DeepCpG applies a similar method using scBS-Seq, DNA sequence, and a bidirectional gated recurrent network [@tag:Angermueller2017].
+Methods like DAPL, MRCNN, and DeepMethyl incorporate both sequence and topological structure [@tag:Qiu2018; @tag:Tian2019; @tag:Khwaja2017; @tag:Wang2016_methyl; @tag:Fu2019].
+In addition, gene expression has been used to infer and impute methylation states [@tag:Peng2019; @tag:Levy-Jurgenson2018], methylation of genes can be predicted from promoter methylation [@tag:Pan2018], and convolutional models have been able to predict methylation status from images [@tag:Momeni2018; @tag:Korfiatis2017].
+While these examples of methylation imputation and inference methods have value, it is imperative to recognize limitations of imputing cytosine modifications.
+Imputing DNA methylation has complexities above and beyond genotype imputation.
+Correlation of DNA methylation marks can depend on cell types and other factors that vary by sample.
+As the number of tissue types and cell types with whole-genome bisulfite sequencing and oxidative bisulfite sequencing grows, the accuracy of DNA methylation imputation is expected to increase.
+While these methods reduce the computational overhead at comparable performance to other popular methylation imputation methods such as k-nearest neighbors, random forests, singular value decomposition, and multiple imputation by chained equations, the software implementations will need to become more user-friendly to gain widespread adoption.  
```

Comment:
```suggestion
While these methods, such as the autoencoder-based DAPL, reduce the computational overhead at comparable performance to other popular methylation imputation methods such as k-nearest neighbors, random forests, singular value decomposition, and multiple imputation by chained equations, the software implementations will need to become more user-friendly to gain widespread adoption [@tag:Qiu2018].  
```

Reply:
@jlevy44 because `[@tag:Qiu2018]` is the DAPL paper, should I move it to be earlier in the sentence?

> While these methods, such as the autoencoder-based DAPL [@tag:Qiu2018], reduce...

Otherwise it looks like the reference is about user-friendliness.

Reply:
Oh I see. That works! Once you commit, I'll give the okay.

Reply:
@agitter

Reply:
@jlevy44 I committed your suggestion and then moved the reference in a separate commit.